samples/apps/copilot-chat-app/webapi/Skills/ChatSkill.cs

```diff
@@ -370,9 +418,10 @@ public async Task<SKContext> ChatAsync(string message, SKContext context)
         }
 
         // Extract semantic memory
+        // TODO: Ask TAO - on Plan approval response, do we need to exclude plan from semantic memory?
```

Comment:
@TaoChenOSU

Reply:
Is there a reason we are trying to avoid storing the plan in semantic memory?

If a user cancels a plan they may then type up a prompt that asks the bot to change the plan in a certain step or how it approaches something over multiple steps. The LLM probably needs the previous plan to know how to modify it.

Reply:
Resolved with Tao offline - will leave as is. It doesn't really matter if the plan is stored in semantic memory or not so we'll leave it to the bot to determine that